<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copyright</class>
 <widget class="QWidget" name="copyright">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_icon">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>选择图标</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_icon">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_project">
       <property name="text">
        <string>项目名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineedit_project"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_version">
       <property name="text">
        <string>版本：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineedit_version">
       <property name="placeholderText">
        <string>最多四位数字版本号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_company">
       <property name="text">
        <string>企业名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineedit_company"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_copyright">
       <property name="text">
        <string>版权：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineedit_copyright"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_trademark">
       <property name="text">
        <string>商标：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineedit_trademark"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
